src/plugins/intel_gpu/src/plugin/transformations/broadcast_reshape_matmul_fusion.cpp

```diff
@@ -10,14 +10,181 @@
 #include "openvino/op/broadcast.hpp"
 #include "openvino/op/constant.hpp"
 #include "openvino/op/reshape.hpp"
+#include "openvino/op/unsqueeze.hpp"
 #include "openvino/pass/pattern/op/wrap_type.hpp"
 #include "openvino/pass/pattern/op/or.hpp"
 #include "transformations/utils/utils.hpp"
 
 namespace ov {
 namespace intel_gpu {
 
+class UnsqueezeBroadcastReshapeMatmulMatcher : public ov::pass::MatcherPass {
+public:
+    OPENVINO_RTTI("UnsqueezeBroadcastReshapeMatmulMatcher", "0");
+    UnsqueezeBroadcastReshapeMatmulMatcher();
```

Comment:
Why did you add a separate matcher for that? It has a little difference with existing one

Reply:
Changed to update by renaming the existing matcher